src/components/List.css

```diff
@@ -13,3 +13,19 @@
 .inactive {
   border-left: 5px solid lightgrey;
 }
+
+.top-container {
```

Comment:
Did some positioning of the `button`, `label`, and `search bar` just so the elements weren't so close together, this can be change to whatever the group decides.

Reply:
👍